.circleci/config.yml

```diff
@@ -163,19 +163,20 @@ jobs:
       - store_artifacts:
           path: _out
 
-  ganache_related_tests:
-    machine:
-      image: ubuntu-1604:201903-01
-    steps:
-      - checkout
-      - run: ./.circleci/install-go.sh
-      - run: ./.circleci/retry.sh 5 go get github.com/orbs-network/go-junit-report
-      - run: ./docker/test/ganache-related-test.sh
-      - store_artifacts:
-          path: _out
-          destination: test-output
-      - store_test_results:
-          path: _reports
+# TODO Ethereum disconnect
```

Comment:
If you must keep the comments here (i would delete them) please give a proper explanation at the top as for what it's there for and when to uncomment it. for example: `uncomment these lines to enable tests which require ganache`